Makefile

```diff
@@ -1,6 +1,7 @@
 .PHONY : entry-point-test
 entry-point-test:
 	python -c "from featuretools import nlp_primitives"
+	python -c "from featuretools.primitives import Elmo"
```

Comment:
```suggestion
	python -c "from featuretools.primitives import Elmo"
	python -c "from nlp_primitives import Elmo"
```

Reply:
Put this in because EvalML calls it like this:
https://github.com/alteryx/evalml/blob/bb0f4f7f6a874916f73d2c8982f014828fe91dd7/evalml/pipelines/components/transformers/preprocessing/natural_language_featurizer.py#L6